godot-core/src/builtin/packed_array.rs

```diff
@@ -503,6 +508,53 @@ macro_rules! impl_packed_array {
     }
 }
 
+// Helper macro to only include specific functions in the code if the Packed*Array provides the function.
+macro_rules! is_packed_byte_array {
+    (false) => {
+        /// Returns a `PackedByteArray` with each value encoded as bytes.
+        pub fn to_byte_array(&self) -> PackedByteArray {
+            self.as_inner().to_byte_array()
+        }
+    };
+    (true) => {
```

Comment:
Instead of `true`/`false`, you could probably use the type name here?
I.e.
```rs
(PackedByteArray) => { ... },
($PackedArray:ident) => { ... }
```

That way, you would not need to introduce a new key `is_packed_byte_array`. Just pass the existing `$PackedArray` key.

Reply:
This is way better. I looked for a solution like this.